cfme/v2v/migrations.py

```diff
@@ -325,6 +331,8 @@ class MigrationDashboardView(BaseLoggedInPage):
 
     @property
     def is_displayed(self):
+        # TODO Uncomment next line, remove line after that
```

Comment:
This will be required to be done after next 5.10z build, as I built this code against upstream and there is a bug on vertical navigation in upstream. Bug should be resolved in next 5.10 build.